LibVLCSharp.WPF/README.md

```diff
@@ -0,0 +1,30 @@
+# LibVLCSharp.WPF
+
+[![NuGet Stats](https://img.shields.io/nuget/v/LibVLCSharp.WPF.svg)](https://www.nuget.org/packages/LibVLCSharp.WPF)
+[![NuGet Stats](https://img.shields.io/nuget/dt/LibVLCSharp.WPF.svg)](https://www.nuget.org/packages/LibVLCSharp.WPF)
+
+LibVLCSharp.WPF is the WPF integration for LibVLCSharp.
+
+It contains the views that allow to display a video played with [LibVLCSharp](../LibVLCSharp/README.md)
```

Comment:
allows?

Reply:
:D I actually wrote _allows_ first and then changed it. I'm not sure... It is the view**s** that allow, no?

Reply:
You're right, I misread. 👍